### PR TITLE
Added Soundcloud Functionality!

### PR DESCRIPTION
**Pull request by andrewyang96**
Changes made:
- "Play Soundcloud" button added beside the "Play Youtube" button.
- You can input a Soundcloud URL to play on Beats!
- Some columns were added, and other columns changed size. I added a migration script to handle that.
- Updated font-awesome from 4.0.3 to 4.4.0. Older releases of font-awesome didn't have the Soundcloud icon.

### CODE REVIEW

**Comment by ajmadsen** [comment]
Can you merge the two buttons together into "Play Stream" and handle the back-end based on what the link is? It would reduce UI clutter and standardize the process for adding more streaming services.

**Comment by andrewyang96** [comment]
OK, I'll take that suggestion in account.

**Comment by kevinwang** [comment]
If you go this route you should also revert 6fe031fc9507c29e46014ceb124aacb575013d4d as we will no longer need to update Font Awesome.

File: soundcloudlib.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import soundcloud
+import json
+
+sc_client = soundcloud.Client(client_id="2f139eb5a94db8e2e6db649070c1da16")
```
Comment (kevinwang) [inline soundcloudlib.py]: Can you put this in beats.cfg? I don't want tokens like this to be hardcoded.

**Comment by andrewyang96** [comment]
OK, I finished the refactoring. There are merge conflicts, though. I think they're in the config files.

**Pull request merged**